src/lib/stores/player.ts

```diff
@@ -19,6 +22,8 @@ export type PlayerStore = {
 	};
 };
 
+let progressTimer: NodeJS.Timer;
```

Comment:
i think timer is the interface of interval. But i think it would be easier to understand if you call this `progressInterval` and also use the `NodeJS.Interval` type